src/calculateRentalCost.js

```diff
@@ -28,7 +28,26 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalPrice = 0;
+  const term = {
+    SHORT: 3,
+    LONG: 7,
+  };
+  const discount = {
+    SMALL: 20,
+    BIG: 50,
+  };
+  const PRICE_PER_DAY = 40;
+
+  totalPrice = days * PRICE_PER_DAY;
+
+  if (days >= term.SHORT && days < term.LONG) {
+    return totalPrice - discount.SMALL;
+  } else if (days >= term.LONG) {
+    return totalPrice - discount.BIG;
+  }
+
+  return totalPrice;
```

Comment:
- don't need to change totalPrice 
- else is redundant cause after return the function will be stoped 
- it's better to user UPPER_CASE for static constant 
```suggestion
  const TERM = {
    SHORT: 3,
    LONG: 7,
  };
  const DISCOUNT = {
    SMALL: 20,
    BIG: 50,
  };
  const PRICE_PER_DAY = 40;
  const totalPrice = days * PRICE_PER_DAY;

  if (days >= TERM.SHORT && days < TERM.LONG) {
    return totalPrice - DISCOUNT.SMALL;
  }
  
  if (days >= TERM.LONG) {
    return totalPrice - DISCOUNT.BIG;
  }
  return totalPrice;
```